code/__DEFINES/is_helpers.dm

```diff
@@ -8,6 +8,8 @@
 
 #define isatom(A) (isloc(A))
 
+#define istom(A) (istype(A, /turf) || istype(A, /obj) || istype(A, /mob))
```

Comment:
The name is incorrect. And this really shouldn't be a define it's buildmode specific restriction.

Reply:
i mean i feel like defining things that players can see and interact with should be useful, but ok.
also nah fam tom like atom but without the area